preprocessor/src/main/java/com/azure/autorest/preprocessor/tranformer/Transformer.java

```diff
@@ -235,11 +235,11 @@ private static boolean nonNullNextLink(Operation operation) {
     return operation.getExtensions().getXmsPageable().getNextLinkName() != null && !operation.getExtensions().getXmsPageable().getNextLinkName().isEmpty();
   }
 
-  private static class PagingNextOperationSignature {
+  private static class OperationSignature {
```

Comment:
Is it just a rename? If this is only for "next operation", the origin name seems more approparate?

Reply:
Yeah, just a rename. I'm more towards the new name. There are two places that use this class:
1. `Map<OperationSignature, Schema> pagingNextOperationResponseSchemaMap` which maps the next page operation to the paging operation's response schema.
2. `Map<OperationSignature, Operation> operationNextPageOperationMap`, added in this PR, maps paging operation to its next page operation.

In the first usage, it's next page operation, while in the second it's not(rather paging operation itself). Guess `OperationSignature` is more general?

Reply:
OK. Seems you've enlarged the usage of this class.

Previously it only created under `operation.getExtensions().getXmsPageable().getOperationName() != null` condition, and it only for the next page operation.

Here in `operationNextPageOperationMap` you seems using it for normal operation as well?

Reply:
Added comment.